e2e/libs/k8s/k8s.py

```diff
@@ -32,6 +32,18 @@ async def restart_kubelet(node_name, downtime_in_sec=10):
 
     delete_pod(pod_name)
 
+def get_longhorn_node_status(node_name):
+    exec_cmd = ["kubectl", "-n", "longhorn-system", "get", "nodes.longhorn.io"]
```

Comment:
Consider using `jsonpath` to retrieve the ready condition. This approach is more reliable because it directly retrieves the specific data, so we don't have to parse based on the column position.
```bash
kubectl -n longhorn-system get nodes.longhorn.io <node_name> -o jsonpath="{.statusditions[?(@.type=='Ready')].status}"
```

Reply:
Updated!
![image](https://github.com/user-attachments/assets/46e5e642-373e-4cb4-b405-e59e75563787)